app/src/main/java/com/telnyx/webrtc/sdk/ui/MainActivity.kt

```diff
@@ -442,12 +451,14 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun disconnectPressed() {
```

Comment:
so this function now is useless?

Reply:
no no this function is not useless

when user wants to disconnect the socket then call this functional